test/options.js

```diff
@@ -0,0 +1,86 @@
+var test = require('tape');
+var Graylog = require('..');
+
+test('options', function (t) {
+	t.throws(function () {
```

Comment:
Recommend that each throw cases be in its own test, and group related throws into describes. This way, if something starts to throw, it should display a bit more nicely.